src/conv.jl

```diff
@@ -188,7 +189,20 @@ for (front_name, backend) in (
         function $(Symbol("$(front_name)!"))(
                         out::AbstractArray{T,5}, in1::AbstractArray{T,5},
                         in2::AbstractArray{T,5}, cdims::ConvDims; kwargs...) where {T <: $G}
-            $(Symbol("$(front_name)_$(backend)!"))(out, in1, in2, cdims; kwargs...)
+            x_cs = Iterators.partition(1:size(in1, 4),
+                                       channels_in(cdims) ÷ groupcount(cdims))
```

Comment:
```suggestion
            x_cs = Iterators.partition(1:channels_in(cdims) , groupcount(cdims))
```
the number of partitions should be equal to the number of groups, right?

Reply:
ops, forget what i said, the second argument of partition is the number of element in each partition